src/client/pages/Offer/Introduction/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
```

Comment:
Not much to see here 👀 

This is a copy of `OfferNew/Introduction/index.tsx` with some small modifications:
- changed the order of the imports to a way I think it's more organized
- rename `Props` --> `IntroductionProps`
- changed component api (`IntroductionProps`) to accept a `quoteCartId`. That value is then provided to `Sidebar`